### ISSUE TITLE
Investigate redundant Samples Check

### ISSUE THREAD

**Issue by Manvi-Agrawal** [microsoft/qsharp#1933]
**Is your feature request related to a problem? Please describe.**
There exists a typescript test that executes samples in samples.mjs ([Code link](https://github.com/microsoft/qsharp/blob/d1fb2a164a3ec4db73506beda3a70e097ddaacc9/npm/qsharp/test/basics.js#L350-#L370) ) . IMO, it would be good to investigate whether it is relevant in the current scenario after this repo has been enhanced for testing efficiency primarily  in #1533 #1797. As per my understanding we are now checking for lint and sample execution for all samples, so it might be a good idea to get rid of redundancy.

**Describe the solution you'd like**
Get rid of redundant test if its no longer required.

**Describe alternatives you've considered**
Dont remove the test if its needed.

**Additional context**
- Earlier, observation raised in https://github.com/microsoft/qsharp/issues/1709 based on Q# infra at the time of raising the issue.
- See [this comment](https://github.com/microsoft/qsharp/pull/1797/files/f37e2d35c2e10de59fa2473590405706c2336492#r1723613762)

**Comment by sezna** [comment]
@Manvi-Agrawal Yep, this seems like a redundancy and removing that JS would be practical.

**Comment by PratikFandade** [comment]
Hi @Manvi-Agrawal @sezna , Can I work on this issue?

**Comment by Manvi-Agrawal** [comment]
@PratikFandade sounds good. This should be fairly straightforward. Let me know if need any help with this.

**Comment by theARCania** [comment]
Hi @Manvi-Agrawal @sezna  Can I take this up if this issue still needs fixing?
